webapp/public/javascripts/controllers.js

```diff
@@ -15,12 +15,13 @@ lunchrControllers.controller('MainPageController', ['$scope', '$http', '$state',
                 return;
             }
 
-            $http.post('/api/users/authenticate', {email: $scope.email, password: $scope.password}).
-                success(function (data, status, headers, config) {
+            $http.post('/api/users/authenticate', {email: $scope.email, password: $scope.password})
+                .success(function (data, status, headers, config) {
```

Comment:
everywhere else it seems the . goes inline with the function, not on the line before. since I was in the file i just moved it.